ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/WriteService.scala

```diff
@@ -103,4 +103,9 @@ trait WriteService
       transaction: SubmittedTransaction,
       estimatedInterpretationCost: Long,
   )(implicit telemetryContext: TelemetryContext): CompletionStage[SubmissionResult]
+
+  /** Indicates whether command deduplication should be enabled when using this [[WriteService]]
+    * This is temporary until we fully transition from [[com.daml.ledger.participant.state.v1.WriteService]] to [[WriteService]]
+    */
+  def isDeduplicationEnabled: Boolean
```

Comment:
As discussed, let's default it to `false`.

We _always_ deduplicate; the question is whether the API should deduplicate, or the driver. So perhaps we can rename this to `isApiDeduplicationEnabled`?

Reply:
Added the false default.
I chose to not add the default so that any implementer would have to be explicit about it to avoid surprises. But if you think it makes sense to have the default I'll push it